src/App.js

```diff
@@ -1,23 +1,22 @@
-import logo from './logo.svg';
-import './App.css';
+import './App.scss';
+import { Route, Switch } from 'react-router-dom';
+import PokeList from './views/PokeList';
+import PokeDetails from './views/PokeDetails';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Switch>
```

Comment:
In general the list should be the home of the app `/`, then details page with ID as you implemented it, and in the end a Route for NotFound case.